src/util.cpp

```diff
@@ -500,24 +500,18 @@ fs::path GetMasternodeConfigFile()
     return AbsPathForConfigVal(pathConfigFile);
 }
 
-void ArgsManager::ReadConfigFile(const std::string& confPath)
+void ArgsManager::ReadConfigStream(std::istream& stream)
 {
-    fs::ifstream streamConfig(GetConfigFile(confPath));
-    if (!streamConfig.good()) {
-        // Create empty pivx.conf if it does not exist
-        FILE* configFile = fsbridge::fopen(GetConfigFile(confPath), "a");
-        if (configFile != NULL)
-            fclose(configFile);
-        return; // Nothing to read, so just return
-    }
+    if (!stream.good())
+        return; // No pivx.conf file is OK
 
     {
         LOCK(cs_args);
         std::set<std::string> setOptions;
         setOptions.insert("*");
 
-        for (boost::program_options::detail::config_file_iterator it(streamConfig, setOptions), end; it != end; ++it) {
-            // Don't overwrite existing settings so command line settings override pivx.conf
+        for (boost::program_options::detail::config_file_iterator it(stream, setOptions), end; it != end; ++it) {
+            // Don't overwrite existing settings so command line settings override bitcoin.conf
```

Comment:
Unneeded change: `pivx.conf`

Reply:
already fixed in 23a4633ff0cb82c8470bf815d73baf601e3653b2, and then removed in 2658771b9471e6b6e445dcd9f7b95077d058f3a2